src/app/qgsmaptoolmovefeature.h

```diff
@@ -23,7 +23,14 @@ class APP_EXPORT QgsMapToolMoveFeature: public QgsMapToolAdvancedDigitizing
 {
     Q_OBJECT
   public:
-    QgsMapToolMoveFeature( QgsMapCanvas* canvas );
+    //!* Mode for moving features
```

Comment:
Does `//!*` work, or should this be `//!`? My gut feeling is that this will put a '*' as the first character in the doc.